src/app/components/navigation-button/navigation-button.component.ts

```diff
@@ -8,4 +18,51 @@ import { Component } from '@angular/core';
   ],
 })
 export class NavigationButtonComponent {
+  constructor(
+    private readonly apollo: Apollo,
+    private readonly router: Router
+  ) {}
+
+  openCheckout(): void {
+    this.apollo.query<me>({ query: getMeQuery }).subscribe(me => {
```

Comment:
`unsubsribe` is missed. You may also use `takeeUntil` pattern: https://github.com/VirtoCommerce/vc-demo-xapi-app/blob/1fc7155549d0574bdd50af51472525809c178bbb/src/app/modules/registration/components/personal-information/personal-information.component.ts#L48